projects/sensory_interface/public/app.js

```diff
@@ -131,4 +146,39 @@ function onCellChange(event) {
     let selectedValue = elementUnderTouch.firstChild.data;
     startSoundPlayback(selectedValue);
     event.stopPropagation();
+}
+
+function onRadioChange(radio) {
+    let minValuePicker = document.getElementById("minValue");
+    let maxValuePicker = document.getElementById("maxValue");
+    if (radio.value == "auto") {
+        minValuePicker.disabled = true;
+        maxValuePicker.disabled = true;
+    } else {
+        minValuePicker.disabled = false;
+        maxValuePicker.disabled = false;
+    }
+    findMinAndMaxValues();
+}
+
+function findMinAndMaxValues() {
+    let input = document.getElementById("dataInput").value;
```

Comment:
Please add a TODO to have a method to parse the string into a data structure (array of arrays, list of lists etc) and use it both here and in `processData()`.